src/pages/index.tsx

```diff
@@ -201,8 +201,18 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 
   const recentTagsByUsers = await getMediaForFeed(20, 3)
 
-  const recentTags = recentTagsByUsers.flatMap(entry => entry.mediaWithTags)
+  const testAreaIds = Array.from(new Set((process.env.NEXT_PUBLIC_TEST_AREA_IDS ?? '').split(',')))
 
+  const mediaWithTags = recentTagsByUsers.flatMap(entry => entry.mediaWithTags)
+
+  const recentTags = mediaWithTags.filter(tag => {
+    return tag.entityTags.some(entityTag => {
+      return testAreaIds.every(testId => {
+        const regex = new RegExp(testId, 'g')
+        return !regex.test(entityTag.ancestors)
+      })
+    })
+  })
```

Comment:
The original issue should be expanded to something like "Prevent Test Area  **and their child areas'** photos from showing on the front page"

Consider this scenario:
1. I tagged a climb under NZ Sport & Trad
2. I want to exclude photos in NZ test area as well as those in child areas
3. I specify NZ test area id in .env

Home / New Zealand / **NZ test area** / NZ Sport & Trad

---

If you inspect each entityTag object, you'll see the `ancestors` field containing a list of ancestor IDs which corresponds to the breadcrumbs: New Zealand / **NZ test area** / NZ Sport & Trad

ancestors: '2d2e7df8-8e0c-5cdf-ab35-42a39b964e5a,**f72326ab-154b-527b-8bf6-4f9b73e7467c**,75be7758-98d9-5dd3-a1f5-6c6590b2d5aa'


```
{
  __typename: 'MediaWithTags',
  id: '64c143eada6d989cb734a886',
  username: 'viet-test-profile112aaabdsbdsb',
  mediaUrl: '/u/b9f8ab3b-e6e5-4467-9adb-65d91c7ebe7c/mQw9tHQtnn.jpeg',
  width: 2000,
  height: 2667,
  uploadTime: 1690387434598,
  entityTags: [
    {
      __typename: 'EntityTag',
      id: '64c143eada6d989cb734a885',
      targetId: '2d63fb47-b2eb-42f1-af9f-26d4b7a1ee5a',
      climbName: 'climb one',
      areaName: 'NZ Sport & Trad',
      ancestors: '2d2e7df8-8e0c-5cdf-ab35-42a39b964e5a,f72326ab-154b-527b-8bf6-4f9b73e7467c,75be7758-98d9-5dd3-a1f5-6c6590b2d5aa',
      type: 0
    }
  ]
}
```


'2d2e7df8-8e0c-5cdf-ab35-42a39b964e5a,**f72326ab-154b-527b-8bf6-4f9b73e7467c**,75be7758-98d9-5dd3-a1f5-6c6590b2d5aa'